tests/shared/src/test/scala-2/zio/schema/DynamicValueSpec.scala

```diff
@@ -100,6 +112,33 @@ object DynamicValueSpec extends ZIOSpecDefault {
             assertTrue(json2 == Right(json))
           }
         } @@ TestAspect.size(250) @@ TestAspect.ignore
+      ),
+      suite("hashCode and equality consistency")(
+        test("hashCode does not change") {
+          val primitive1 = DynamicValue.Primitive(123, StandardType.IntType)
```

Comment:
Rather than these tests, which test concrete values, you should use a generator for dynamic value `Gen`. I am not sure if one exists, but if not, you can create one pretty easily.

Reply:
Thanks! Resolved in the next revision.